central/graphql/resolvers/namespaces.go

```diff
@@ -651,6 +654,24 @@ func (resolver *namespaceResolver) DeploymentCount(ctx context.Context, args Raw
 	return resolver.root.DeploymentCount(ctx, RawQuery{Query: &query})
 }
 
+func (resolver *namespaceResolver) NetworkPolicyCount(ctx context.Context, args RawQuery) (int32, error) {
+	defer metrics.SetGraphQLOperationDurationTime(time.Now(), pkgMetrics.Namespaces, "NetworkPolicyCount")
+	if err := readSecrets(ctx); err != nil {
```

Comment:
wrong one

Reply:
Whoops. Switched to `resources.NetworkPolicy`